src/tests/ftest/util/run_utils.py

```diff
@@ -15,38 +15,40 @@ class RunException(Exception):
     """Base exception for this module."""
 
 
-class RemoteCommandResult():
-    """Stores the command result from a Task object."""
+class ResultData():
```

Comment:
FYI - I think you could have also referenced this class externally like:
`RemoteCommandResult.ResultData`. But this change makes sense anyway

Reply:
I could have, but I think its a bit cleaner to have it defined independently.